lib/rubocop/cop/style/redundant_line_continuations.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Avoid redundant line continuations.
+      #
+      # @example
+      #   # bad
+      #   foo. \
+      #     bar
+      #   foo \
+      #     .bar \
+      #       .baz
+      #   foo&. \
+      #     bar
+      #   [foo, \
+      #     bar]
+      #   {foo: \
+      #     bar}
+      #   foo(bar, \
+      #     baz)
+      #
+      #   # good
+      #   foo.
+      #     bar
+      #   foo
+      #     .bar
+      #       .baz
+      #   foo&.
+      #     bar
+      #   [foo,
+      #     bar]
+      #   {foo:
+      #     bar}
+      #   foo(bar,
+      #     baz)
+      #   foo('bar' \
+      #     'baz')
+      #
+      class RedundantLineContinuations < Base
```

Comment:
I'd change the name to be in singular form - "RedundantLineContinuation". I think that's how we typically name cops.

Reply:
Indeed it is. I changed it to `RedundantLineContinuation`.